sql/moz-fx-data-shared-prod/mozilla_vpn_derived/all_subscriptions_v1/query.sql

```diff
@@ -59,11 +46,9 @@ stripe_invoice_lines AS (
   SELECT
     lines.subscription AS subscription_id,
     IF(
-      -- see https://bugzilla.mozilla.org/show_bug.cgi?id=1712027#c1
-      -- for why presence of "paypalTransactionId" alone is insufficient
-      "paypalTransactionId" IN (SELECT key FROM UNNEST(invoices_v1.metadata))
-      OR (invoices_v1.charge IS NULL AND invoices_v1.status = "paid"),
-      STRUCT("Paypal" AS provider, paypal_country.country),
+      "paypalTransactionId" IN (SELECT key FROM UNNEST(invoices_v1.metadata)),
```

Comment:
as of #2209 `paypalTransactionId` is sufficient for accurately identifying paypal invoices.